CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 CORE
 ----
+
```

Comment:
No need to add changelog, i'm adding tags when i merge PRs so we can autogenerate it later. No need to remove it from this one either :) just so you know for next time